<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="qtLandmarksWidget">
       <attribute name="title">
        <string>Landmarks</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="qtMapWidget">
       <attribute name="title">
        <string>Map View</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1173</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="qtConvertToLandmarksAction"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="qtLoadMapsAction"/>
    <addaction name="qtLoadWaypointsAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_LoadWaypoints">
   <property name="text">
    <string>Load waypoints/GPS</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_SaveGPS">
   <property name="text">
    <string>Save as GPS data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_SaveCSV">
   <property name="text">
    <string>Save as CSV data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="qtConvertToLandmarksAction">
   <property name="text">
    <string>Convert to Landmarks</string>
   </property>
  </action>
  <action name="qtLoadWaypointsAction">
   <property name="text">
    <string>Load Waypoints</string>
   </property>
  </action>
  <action name="qtLoadMapsAction">
   <property name="text">
    <string>Load Map</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
